src/core/DataTable/exportCSV.js

```diff
@@ -0,0 +1,37 @@
+function convertToCSV(objArray) {
+  return (typeof objArray !== 'object' ? JSON.parse(objArray) : objArray)
+    .map(innerArray => innerArray.filter(val => val).join(','))
```

Comment:
Looks like innerArray is not always an array:
![Screenshot from 2020-02-04 17-03-43](https://user-images.githubusercontent.com/1779590/73751539-5b2ff400-4770-11ea-8331-d06e99dfd4a4.png)